tools_for_todoist/services/calendar_to_todoist.py

```diff
@@ -192,10 +199,11 @@ def _todoist_sync(self):
                     continue
 
                 last_completed_date = calendar_event.next_occurrence(event_date)
-                if not is_allday(last_completed_date):
-                    last_completed_date = last_completed_date.astimezone(UTC)
+                if last_completed_date is not None:
+                    if not is_allday(last_completed_date):
+                        last_completed_date = last_completed_date.astimezone(UTC)
+                    calendar_event.save_private_info(CALENDAR_LAST_COMPLETED, last_completed_date)
 
-                calendar_event.save_private_info(CALENDAR_LAST_COMPLETED, last_completed_date)
                 calendar_event.save()
```

Comment:
we should only save the calendar event if we modified it (so `save_private_info` goes together with this line)